cabal.project

```diff
@@ -41,3 +41,6 @@ benchmarks: True
 
 -- Always show detailed output for tests
 test-show-details: direct
+
+allow-newer: base
+allow-newer: plutus-ledger-api
```

Comment:
Which packages do need these constraints? There is a nice syntax here to provide more context. I think it goes like: `allow-newer: somepkg:base` to say that `somepkg` needs to have its upper bounds removed for `base`

Reply:
We don't this is is noise.

Reply:
This is noise from ghc 9.10. Not relevant to this PR.